cmd/server/grpc/main.go

```diff
@@ -38,11 +39,18 @@ var (
 	errNoPeer = errors.New("no peer exists")
 )
 
-type server struct {
-	pb.UnimplementedSFUServer
+// Server struct
+type Server struct {
```

Comment:
can we keep this private?

do we no longer need `pb.UnimplementedSFUServer`?

are you sure we need to change this?

Reply:
when I compiled the .proto, the line was removed.

Reply:
golang lint was making noise because of the server struct being private